Added command line option for enabling tearing for DX11

stefano-bonifazi opened the pull request.

stefano-bonifazi committed:
Added command line option for enabling tearing for DX11

JeffersonMontgomery-Intel commented on src/WinWrapper.cpp:
```diff
@@ -331,6 +335,15 @@ int main(int argc, char** argv)
+    {
+        if (gSettings.vsync)
+        {
+            std::cout << "Tearing cannot be used when vsync is on. Disabling it!" << std::endl;
```
Can you tweak this to make it clear you're disabling allowTearing and not disabling vsync?

stefano-bonifazi replied:
done

stefano-bonifazi committed:
Making it more clear that we disable tearing when vsync is on.

stefano-bonifazi committed:
Added allow tearing to D3D12

Lowered default number of Asteroids to appreciate better the effect of
Tearing.

stefano-bonifazi committed:
Added check for tearing support
